test/runtime/Policies.go

```diff
@@ -951,22 +952,40 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 	Context("TestsEgressToHost", func() {
 		hostDockerContainer := "hostDockerContainer"
 		hostIP := "10.0.2.15"
+		otherHostIP := ""
 
 		BeforeAll(func() {
 			By("Starting httpd server using host networking")
 			res := vm.ContainerCreate(hostDockerContainer, helpers.HttpdImage, helpers.HostDockerNetwork, "-l id.hostDockerContainer")
 			res.ExpectSuccess("unable to start Docker container with host networking")
+
+			By("Detecting host IP in world CIDR")
+			res = vm.Exec("ip route | grep '192.168' | head -n 1")
+			res.ExpectSuccess("unable to detect host IP for CIDR test")
+			routeParams := strings.Split(res.Output().String(), " ")
+			for _, param := range routeParams {
+				if net.ParseIP(param) != nil {
+					otherHostIP = param
+					break
+				}
+			}
+			Expect(otherHostIP).To(Not(Equal("")))
+			By(fmt.Sprintf("Using %s for world CIDR IP", otherHostIP))
```

Comment:
I would suggest to delete this one, `otherHostIP` is in other 3 debug messages, don't add a real value for me this one ;-)

Reply:
This message is stating what the IP address is, the other logs don't say that. I think it would be easier when debugging to see something like:

* Detecting host IP in world CIDR
* Using 192.0.2.3 for world CIDR IP
* Pinging 192.0.02.3 from app1 before importing policy
* ...